newt/image/image.go

```diff
@@ -46,6 +46,10 @@ import (
 	"mynewt.apache.org/newt/util"
 )
 
+// Set this to enable RSA-PSS for RSA signatures, instead of PKCS#1
+// v1.5.  Eventually, this should be the default.
+var UseRsaPss = false
```

Comment:
This would look better if this setting was a property of Image, as that's where all the other data associated with the signing process is being collected.